docs/installation/local.md

```diff
@@ -2,169 +2,177 @@
 ```sh
-git clone -b development https://github.com/fossasia/open-event-server.git
+git clone https://github.com/fossasia/open-event-server.git
 cd open-event-server
 ```
-**Note :** If you want to contribute, first fork the original repository and clone the forked repository into your local machine followed by ```cd``` into the directory
+
+- If you want to contribute, first fork the original repository to your GitHub profile and clone that fork into your local machine, followed by `cd` into the directory
+
 ```sh
-git clone https://github.com/USERNAME/open-event-server.git
+git clone git@github.com:USERNAME/open-event-server.git
 cd open-event-server
 ```
 
-* **Step 1** - Install python3 requirements. You need to be present in the root directory of the project.
+- Tip:
 
-# System Wide Installation
+  + Setup SSH key in your profile, and use SSH method to clone the source code, so that you don't have to type password repeatly.
 
-```sh
-sudo -H pip3 install -r requirements.txt
-```
-hint: You may need to upgrade your pip version and install following packages if you encounter errors while installing the requirements.
+  + To let your personal fork up-to-date with the original FOSSASIA repo, add original repo to "remote" list and regularly fetch its new content.
 
-**Note:** For Mac OS Sierra users, if you get an error that 'openssl/aes.h' could not be found when installing requirements.txt using pip, try the steps shown here - [OSX openssl header error](https://tutorials.technology/solved_errors/1-OSX-openssl_opensslv_h-file-not-found.html)
+  ```sh
+  git remote add upstream https://github.com/fossasia/open-event-server.git
+  git fetch upstream
+  git merge upstream/development
+  ```
 
-# Installation in Virtual Environment
+## Install system dependencies
 
-You can use either **pip** or **pipenv** to install Open Event Server in a virtual environment.
+These are softwares on which our Open Event server depends, and C-based libraries on which our Python packages depend.
 
-Firstly, open a terminal and enter
+### For Mac OS
 
 ```sh
-# For linux users
-sudo apt-get install python3-dev
-sudo apt-get install libpq-dev
-sudo apt-get install libffi6 libffi-dev
-
-# For macOS users
+brew install postgresql
 brew install python@3
 brew install libmagic
+brew install redis
 ```
 
-## Using pip and virtualenv
+**Note:** For Mac OS Sierra users, if you get an error that 'openssl/aes.h' could not be found when installing requirements.txt using pip, try the steps shown here - [OSX openssl header error](https://tutorials.technology/solved_errors/1-OSX-openssl_opensslv_h-file-not-found.html)
+
+### For Debian/Ubuntu
 
-Open a terminal and enter the following commands to setup a virtual environment
+Enter the project source folder and run:
 
 ```sh
-sudo apt-get install python3.7
-virtualenv -p python3.7 venv
-. venv/bin/activate
+xargs -a deb-packages.txt sudo apt install
```

Comment:
I think we should keep the dependencies explicitly listed here to avoid indirection and it's easier to track updates as well

Reply:
But this `deb-packages.txt` will help deploy process, it allows deploy script to automatically install new dependency if it is added.

Reply:
I've just added a note to README, to let developer know that the list of dependencies are in that file.

Reply:
Since we use docker alpine to build the image, it doesn't help us in deployment particularly. Where else is it being used to install the dependencies?

Reply:
@iamareebjamal If you write down:

```sh
sudo apt install libpq-dev
brew install python@3
```

as before this PR, does it help you build Alpine Docker container?

The old README doesn't help you build Alpine Docker container, why do you use Alpine as excuse to refuse this change?

Reply:
> But this deb-packages.txt will help deploy process, it allows deploy script to automatically install new dependency if it is added.

I was commenting about this. Since we use alpine builds, debian dependencies don't help us in deployment process. Never said that the current docs help us in deployment process. Since, a separate file does not help us in the deployment process, why do we need to add an extra file and an indirection. Any change should have a net positive impact in the maintenance and developer experience.

Reply:
@iamareebjamal 

> debian dependencies don't help us in deployment process

This PR doesn't interfere the deployment process. It is addressing the "local installation guide". Before this PR, the documentation is guiding beginner to setup development environment on Ubuntu/Debian, and this PR still focus on it.
If you want to improve deployment process, let's do it in separate PR.

Reply:
> add an extra file and an indirection

That extra file to help developer install dependencies in one command, and also, if the dependencies list grows overtime, it just make that file row instead of this document. If you concern about list of packages stay in separate file, why don't you concern the _requirements.txt_, which is also a separate file to list dependencies?